### ISSUE TITLE
"Schedule an appointment online" CTA on VAMC Facility Health Service should be styled as link not button

### ISSUE THREAD

**Issue by davidmpickett** [department-of-veterans-affairs/va.gov-cms#15601]
## Describe the defect
Similar to #14928, this is another instance on the VAMC product where a CTA link is styled to visually appear as a button. This creates an accessibility issue as it goes against material honesty. This means that the element (the link) should always look like what it is (a link). To resolve this issue and to align with the VA Design System, this "button" should be updated to be [a secondary action link](https://design.va.gov/components/link/action#secondary).

- Facility service accordion [redesign per VBA](https://www.sketch.com/s/891d33ae-152d-471c-ab5d-9aedf89cf6ff/p/9BAF00BD-1CA9-4FCF-A14B-E2B3D31A18E6/canvas)

## To Reproduce
1. Go to [Sacramento VA Medical Center](https://www.va.gov/northern-california-health-care/locations/sacramento-va-medical-center/#primary-care)
2. Scroll down to **Health services offered here**
4. Open the **Primary Care** accordion
5. See the **Schedule an appointment online** CTA

### Current 

![Screenshot 2023-10-04 180935](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/73cfcf7f-9a24-4976-a91a-aea54efcb90b)

### Expected

![image](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/106678594/1fb3e567-6177-4503-a42c-b6dff32162a0)

## AC / Expected behavior
- [ ] "Schedule an appointment online" CTA is styled as [a secondary action link](https://design.va.gov/components/link/action#secondary) according to the Design system
- [ ] The "schedule an appointment online" link only shows up on VAMC Facility Services where the editor has set "Is online scheduling available for this service?" to Yes
- [ ] Design Review
- [ ] Accessibility review

**Comment by davidmpickett** [comment]
Noting that this has been resolved as part of the Service Location work and is behaving correctly on Tugboat. 

![Screenshot 2024-03-28 at 1 55 38 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/eb8db855-d809-41dd-b89a-22e499d437b4)

Moving this to complete pending integration since there is no additional code work to complete, but the defect is still on prod. Does that make sense @laflannery

**Comment by davidmpickett** [comment]
Verified in prod
![Screenshot 2024-05-15 at 1 12 43 PM](https://github.com/department-of-veterans-affairs/va.gov-cms/assets/51967950/925e4fdc-fa42-4263-bce2-4e3842f33f4a)
